apps/taxes/src/modules/avatax/avatax-connection-schema.ts

```diff
@@ -28,7 +28,7 @@ export type AvataxConfig = z.infer<typeof avataxConfigSchema>;
 export const defaultAvataxConfig: AvataxConfig = {
   name: "",
   companyCode: "",
-  isSandbox: true,
+  isSandbox: false,
```

Comment:
why do you change it now?

Reply:
Tbh, I am not sure how this `true` got here. I didn't notice it because there was a bug with the `AppToggle` value, hence the fix.